cmd/acraserver/acraserver.go

```diff
@@ -148,6 +149,12 @@ func main() {
 		os.Exit(1)
 	}
 
+	if err := config.SetFirewall(*firewallConfig); err != nil {
```

Comment:
I think we shouldn't return error if no config file.
users should be able not to start firewall if they don't want to.

i would check if `firewallConfig` exists first, then `setFirewall`